### PR TITLE
Add plannable icon for assessment_request

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-14022
affects: Parent
release note: none

Test
Go to oct 7 2017 and make sure there is a proper icon for peer reviews plannables (twilson ; observer1)

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/o67lp7p) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fo67lp7p)
teacher: [Link](https://rebrand.ly/8y3glcg) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F8y3glcg)
parent: [Link](https://rebrand.ly/tbnxren) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Ftbnxren)

File: Core/CoreTests/Planner/PlannableTests.swift [hunk 1]
```diff
@@ -67,5 +67,8 @@ class PlannableTests: CoreTestCase {
 
         p = Plannable.make(from: .make(plannable_type: "calendar_event"))
         XCTAssertEqual(p.icon(), UIImage.icon(.calendarMonth, .line))
+
```
Comment (inst-danger) [inline Core/CoreTests/Planner/PlannableTests.swift]:
```suggestion

```

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1023322&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1023323&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Parent

[MBL-14022](https://instructure.atlassian.net/browse/MBL-14022)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.53% | 93.09% | 0.44%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**